04_environment/index.tex

```diff
@@ -20,41 +20,32 @@ \subsubsection{Modifications to the Original concepts}
     \item Instead of a specified return, probabilistic return has been implemented which adds a randomness element to the game to prevent pattern recognition. These probabilities can be tied to the population, disasters, forecasting and number of animals hunted.  
 \end{enumerate}
 
-\subsubsection{Tier System}
-
-The deer hunting and fishing is divided into tiers (the number of fish that can be caught in a given day) which depends on the total input resources. Where the “zero” tier represents the cost of getting to the hunting location.
+Furthermore, the dynamics of our agents and allowing for a more complex decision-making interaction between islands and the environment has been explored. It is worth mentioning that some of these design ideas were implemented in the coursework. These changes have been made to make foraging more volatile and unpredictable, adding to the complexity of the system, allowing the teams to further evaluate the agents’ performance on observing and to use learned knowledge to take action.
 
-Beyond the zero tier, the cost of catching another deer/fish will decrease as it is easier to catch the second animal than the first since the agents have already arrived at the hunting location. It should be noted there is a limit to how many animals the agents can catch in a given day.
+\subsubsection{Tier System}
 
-The focus of fishing is to avoid risk at the expense of payoff. This means that all tier requirements are lower than the deers. However, the return is also lower and in addition, the “zero” tier is also removed. Thus, if only 1 island goes fishing they are still expected to have a low return as long as they have enough resources to catch the first fish.
+The deer hunting and fishing is divided into tiers, which depend on the total input resources. Tiers represent the number of fish that can be caught in a given day and the “zero” tier represents the cost associated with getting to the hunting location. Beyond the zero tier, the cost of catching another deer or fish will decrease as it is easier to catch the second animal than the first since the agents have already arrived at the hunting location. It should be noted there is a limit to how many animals the agents can catch in a given day.
```

Comment:
total input resources ---> total amount of input resources.

**,** and the "zero" tier. ---> do NOT forget the comma

revise the sentence 
"Beyond the zero tier, the cost of catching another deer or fish will decrease as it is easier to catch the second animal than the first **one** . "

I don't understand what you mean by 
"the agents have already arrived at the hunting location" ???

Reply:
Hey,
the use of the term "hunting Location" was used for it to be easier to visualise the situation, I have added an additional sentence to make it clearer.

Reply:
ok thanks, I don't think i see that sentence in this  version :(